parlai/torchscript/modules.py

```diff
@@ -481,7 +488,41 @@ def encode(self, text: str) -> List[str]:
         """
         if self.add_prefix_space:
             text = f' {text}'
-        return self.helper_encode(text)
+
+        # constants for readability
```

Comment:
hmm perhaps it'd help to have a 1-sentence comment about how the special tokens code works at a high level, and maybe also what FINAL and SPLITABLE are?